WooCommerce/WooCommerceTests/ViewRelated/Products/ProductsTabProductViewModelTests.swift

```diff
@@ -93,6 +93,35 @@ final class ProductsTabProductViewModelTests: XCTestCase {
         XCTAssertTrue(detailsText.contains(expectedStockDetail))
     }
 
+    func test_details_contain_sku_when_isSKUShown_is_true() {
+        // Given
+        let sku = "pear"
+        let product = productMock(name: "Yay").copy(sku: sku)
+
+        // When
+        let viewModel = ProductsTabProductViewModel(product: product, isSKUShown: true)
+        let detailsText = viewModel.detailsAttributedString.string
+
+        // Then
+        let skuFormat = NSLocalizedString("SKU: %1$@", comment: "Label about the SKU of a product in the product list. Reads, `SKU: productSku`")
```

Comment:
Super nit: should we remove this localized string to an extension since it's used in more than one test case?

Reply:
Makes sense, updated in https://github.com/woocommerce/woocommerce-ios/pull/7781/commits/cf12627d6e72e59300254895dcc412ef927a77d8